.github/contributing.md

```diff
@@ -0,0 +1,78 @@
+# `too-many-hooks` contributing guide
+
+Thanks for taking the time to contribute to `too-many-hooks`! Any merged PRs will be reflected in the next release of the package and on our storybook immediately.
+
+In this guide you will get an overview of the contribution workflow from opening an issue, creating a PR, reviewing, and merging the PR.
+
+## New contributor guide
+
+To get an overview of the project, read the [README](README.md). Here are some resources to help you get started with open source contributions:
+
+- [Finding ways to contribute to open source on GitHub](https://docs.github.com/en/get-started/exploring-projects-on-github/finding-ways-to-contribute-to-open-source-on-github)
+- [Set up Git](https://docs.github.com/en/get-started/quickstart/set-up-git)
+- [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow)
+- [Collaborating with pull requests](https://docs.github.com/en/github/collaborating-with-pull-requests)
+
+## Getting started
+
+Anyone can open an issue or PR so if there is something you'd like to see added to `too-many-hooks` feel free to open an issue, PR, or both according to the guidelines below
```

Comment:
"open an issue or PR" is a little redundant here. Could be cleaned up to the following:

```suggestion
Anyone can contribute, so if there is something you'd like to see added to `too-many-hooks`, feel free to open an issue, pull request, or both, according to the guidelines below.
```